doc/api/api_changes.rst

```diff
@@ -269,36 +220,148 @@ new and old scaling::
 
 
 
-Default behavior of log scales changed to mask <= 0 values
-----------------------------------------------------------
 
-Calling `matplotlib.axes.Axes.set_xscale` or `matplotlib.axes.Axes.set_yscale`
-now uses 'mask' as the default method to handle invalid values (as opposed to
-'clip'). This means that any values <= 0 on a log scale will not be shown.
 
-Previously they were clipped to a very small number and shown.
+Change to signatures of :meth:`~matplotlib.axes.Axes.bar` & :meth:`~matplotlib.axes.Axes.barh`
+----------------------------------------------------------------------------------------------
+
+For 2.0 the :ref:`default value of *align* <barbarh_align>` changed to
+``'center'``.  However this caused the signature of
+:meth:`~matplotlib.axes.Axes.bar` and
+:meth:`~matplotlib.axes.Axes.barh` to be misleading as the first parameters were
+still *left* and *bottom* respectively::
+
+  bar(left, height, *, align='center', **kwargs)
+  barh(bottom, width, *, align='center', **kwargs)
+
+despite behaving as the center in both cases. The methods now take ``*args, **kwargs``
+is input and are documented to have the primary signatures of::
+
+  bar(x, height, *, align='center', **kwargs)
+  barh(y, width, *, align='center', **kwargs)
+
+Passing *left* and *bottom* as keyword arguments to
+:meth:`~matplotlib.axes.Axes.bar` and
+:meth:`~matplotlib.axes.Axes.barh` respectively will warn.
+Support will be removed in Matplotlib 3.0.
+
 
 Font cache as json
 ------------------
 
 The font cache is now saved as json, rather than a pickle.
 
-Code Removal
+
+Invalid (Non-finite) Axis Limit Error
+-------------------------------------
+
+When using :func:`~matplotlib.axes.Axes.set_xlim` and
+:func:`~matplotlib.axes.Axes.set_ylim`, passing non-finite values now
+results in a ``ValueError``. The previous behavior resulted in the
+limits being erroneously reset to ``(-0.001, 0.001)``.
+
+``scatter`` and ``Collection`` offsets are no longer implicitly flattened
+-------------------------------------------------------------------------
+
+`~matplotlib.collections.Collection` (and thus both 2D
+`~matplotlib.axes.Axes.scatter` and 3D
+`~mpl_toolkits.mplot3d.axes3d.Axes3D.scatter`) no
+longer implicitly flattens its offsets.  As a consequence, ``scatter``'s ``x``
+and ``y`` arguments can no longer be 2+-dimensional arrays.
+
+Deprecations
 ------------
 
-matplotlib.delaunay
-~~~~~~~~~~~~~~~~~~~
-Remove the delaunay triangulation code which is now handled by Qhull
-via ``matplotlib.tri``
+``GraphicsContextBase``\'s ``linestyle`` property.
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+The ``GraphicsContextBase.get_linestyle`` and
+``GraphicsContextBase.set_linestyle`` methods, which effectively had
+no effect, have been deprecated.
+
+
+``NavigationToolbar2.dynamic_update``
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Use :meth:`matplotlib.backend_bases.FigureCanvas.draw_idle` instead.
```

Comment:
I think it's `FigureCanvasBase`, but I'm not sure if that's where one wants to be linking.